app/services/catalog_provider_sqlite.py

```
@@ -0,0 +1,190 @@
+SQLite catalog provider (PR-7).
+
+RU: Провайдер каталога на основе SQLite (read-only).
+EN: SQLite-based catalog provider (read-only).
+
+This provider reads from pre-built SQLite snapshots (no I/O in request-path).
+"""
+
+from __future__ import annotations
+
+import sqlite3
+from dataclasses import asdict
```

Flake8 (7.3.0)

[error] 'dataclasses.asdict' imported but unused

(F401)